imports/plugins/included/sitemap-generator/server/no-meteor/schemas/schema.graphql

```diff
@@ -0,0 +1,19 @@
+extend type Mutation {
+  "Generate sitemap documents"
+  generateSitemaps(input: GenerateSitemapsInput): GenerateSitemapsPayload
+}
+
+"Input for the `generateSitemaps` mutation"
+input GenerateSitemapsInput {
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+}
+
+"Response for the `generateSitemaps` mutation"
+type GenerateSitemapsPayload {
+  "Whether the sitemap generation job was successfully schedule"
+  wasJobScheduled: Boolean
```

Comment:
- Last word in docs string should be "scheduled"
- Update to `Boolean!` if this will never be `null` in the response
- Alphabetize fields